snap/snapcraft.yaml

```diff
@@ -12,6 +12,10 @@ grade: stable
 confinement: strict
 base: core18
 
+system-usernames:
+  passthrough:
```

Comment:
ah I just proposed the inverse usage of passthrough on your other branch, but both should work I think